src/dbi.rs

```diff
@@ -364,25 +373,25 @@ impl From<u16> for MachineType {
 /// `struct SC` in Microsoft's code:
 /// https://github.com/Microsoft/microsoft-pdb/blob/082c5290e5aff028ae84e43affa8be717aa7af73/PDB/include/dbicommon.h#L42
 #[derive(Debug, Copy, Clone)]
-pub(crate) struct DBISectionContribution {
+pub struct DBISectionContribution {
     /// The index of the section.
-    section: u16,
+    pub section: u16,
```

Comment:
Without having looked at the details, it looks like the `section` and `offset` attributes comprise a `PdbInternalSectionOffset`. Since this is now public API, can we parse into that type rather than the raw values?